.github/workflows/release-postman.yml

```diff
@@ -11,11 +11,50 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Build and upload Collection
+
+      - name: Fetch Collection
+        working-directory: ./tools/postman
+        run: |
+          make fetch_openapi
+
+      - name: Compute checksum for current OpenAPI Specification
+        working-directory: ./tools/postman/openapi
+        run: |
+          checksum=$(sha256sum atlas-api.json | cut -d ' ' -f 1)
+          echo "checksum=$checksum" >> "$GITHUB_ENV"
+
+      - name: Check if checksum is saved in the cache
+        id: check-cache
+        uses: actions/cache/restore@v4
+        with:
+          path: ./tools/postman/openapi/openapi-checksum.txt
+          key: postman-openapi-${{ env.checksum }}
```

Comment:
All of those can be part of the fetch. Once openapi files will be saved in the repository all the logic can be removed (including fetch logic)